wildfly/connectors/amazon/feature-pack-amazon/src/main/resources/modules/system/layers/dv/org/jboss/teiid/translator/simpledb/main/module.xml

```diff
@@ -14,6 +14,6 @@
         <module name="javax.resource.api" />
         <module name="org.jboss.teiid.common-core" />
         <module name="org.jboss.teiid.api" />
-        <module name="org.jboss.teiid.translator.simpledb.api" />
+        <module name="org.jboss.teiid.translator.simpledb" />
```

Comment:
This line can be removed as it's now just declaring a dependency on itself.

Reply:
I'll update it first thing in the morning. 
Turned the laptop off just now.